data/json/hobbies.json

```
@@ -91,5 +94,368 @@
+    "points": 2,
+    "skills": [ { "level": 1, "name": "melee" }, { "level": 3, "name": "unarmed" }, { "level": 2, "name": "dodge" } ]
+  },
+  {
+    "type": "profession",
+    "subtype": "hobby",
+    "id": "knife_throwing",
+    "name": "Knife Throwing",
+    "description": "Throwing knives are fun and deadly in the right hands.",
+    "points": 2,
+    "skills": [ { "level": 3, "name": "throw" } ]
+  },
+  {
+    "type": "profession",
+    "subtype": "hobby",
+    "id": "sewing",
+    "name": "Sewing",
+    "description": "You used to sew torn clothes back together and even made your own clothes a few times.",
+    "points": 2,
+    "proficiencies": [ "prof_closures", "prof_elastics" ],
+    "skills": [ { "level": 3, "name": "tailor" }, { "level": 1, "name": "fabrication" } ]
+  },
+  {
+    "type": "profession",
+    "subtype": "hobby",
+    "id": "fencing",
+    "name": "Fencing",
+    "description": "You were in a fencing club, and you weren't too shabby with a saber.",
+    "points": 2,
+    "skills": [ { "level": 3, "name": "stabbing" }, { "level": 1, "name": "melee" } ],
+    "traits": [ "MARTIAL_FENCING" ]
+  },
+  {
+    "type": "profession",
+    "subtype": "hobby",
+    "id": "car_fan",
+    "name": "Gearhead",
+    "description": "You're a big fan of cars and other things that go vroom, whether it's fixing them or driving them.  You may not be a professional, but you know your way around an engine.",
+    "points": 1,
+    "skills": [ { "level": 3, "name": "driving" }, { "level": 1, "name": "mechanics" } ]
+  },
+  {
+    "type": "profession",
+    "subtype": "hobby",
+    "id": "martial_training",
+    "name": "Martial Training",
+    "description": "You have received some martial arts training at a local dojo.  You start with your choice of Karate, Judo, Aikido, Tai Chi, Taekwondo, or Pankration.",
+    "points": 4,
+    "skills": [ { "level": 4, "name": "melee" }, { "level": 4, "name": "unarmed" } ],
+    "traits": [ "MARTIAL_ARTS" ]
+  },
+  {
+    "type": "profession",
+    "subtype": "hobby",
+    "id": "self_defense",
+    "name": "Self-Defense Classes",
+    "description": "You have taken some self-defense classes at a nearby gym.  You start with your choice of Boxing, Capoeira, Krav Maga, Muay Thai, Ninjutsu, Wing Chun, or Zui Quan.",
+    "points": 4,
+    "skills": [ { "level": 4, "name": "melee" }, { "level": 4, "name": "unarmed" } ],
+    "traits": [ "MARTIAL_ARTS2" ]
+  },
+  {
+    "type": "profession",
+    "subtype": "hobby",
+    "id": "shaolin",
+    "name": "Shaolin Adept",
+    "description": "You have studied the arts of the Shaolin monks.  You start with one of the five animal fighting styles: Tiger, Crane, Leopard, Snake, or Dragon.",
+    "points": 4,
+    "skills": [
+      { "level": 4, "name": "melee" },
+      { "level": 3, "name": "bashing" },
+      { "level": 3, "name": "stabbing" },
+      { "level": 3, "name": "cutting" }
```

```suggestion
      { "level": 3, "name": "unarmed" },
      { "level": 3, "name": "dodge" }
```
These are all unarmed styles 😛